types/atom.js.flow

```diff
@@ -1535,6 +1535,7 @@ declare class atom$GrammarRegistry {
   removeGrammarForScopeName(scopeName: string): ?atom$Grammar,
   loadGrammarSync(grammarPath: string): atom$Grammar,
   selectGrammar(filePath: string, fileContents: string): atom$Grammar,
+  assignLanguageMode(bufferOrEditor: atom$TextBuffer | atom$TextEditor, languageId:string): boolean,
```

Comment:
I would appreciate if you fix this: just by adding a spece.
![image](https://user-images.githubusercontent.com/40514306/61971418-fbff6300-b019-11e9-890f-f153e6590808.png)